crates/cdk/src/nuts/nut00/mod.rs

```diff
@@ -373,46 +373,71 @@ pub enum CurrencyUnit {
     Usd,
     /// Euro
     Eur,
+    /// Custom currency unit (3 characters)
+    Custom([u8; 3]),
 }
```

Comment:
We can't assume units are 3 letter codes. So think we need to make this a string and remove Copy